templates/taxi/car_confirm_delete.html

```diff
@@ -0,0 +1,10 @@
+{% extends "base.html" %}
+
+{% block content %}
+  <h2>Are you sure you want to delete car?</h2>
+
+  <form method="post">
+    {% csrf_token %}
+    <button type="submit" class="btn btn-danger">Delete</button>
```

Comment:
Add Cancel button apart from Delete one. The Cancel button will lead to the previous page the user was on.